core/modules/upgraders/system.js

```diff
@@ -13,7 +13,7 @@ Upgrader module that suppresses certain system tiddlers that shouldn't be import
 "use strict";
 
 var DONT_IMPORT_LIST = ["$:/core","$:/Import"],
-	DISABLE_SYSTEM_LIST = ["$:/temp/","$:/state/","$:/StoryList","$:/HistoryList"],
+	DISABLE_PREFIX_LIST = ["$:/temp/","$:/state/","$:/StoryList","$:/HistoryList"],
```

Comment:
@saqimtiaz ... The real name should be DISABLE_PREFIX_LIST. At the moment it's only system tiddlers that are disabled by default. ... But it can be any prefix. So I think that's the right name now.